src/index.ts

```diff
@@ -45,6 +46,7 @@ const commandModules = [
         c.version(require("../package.json").version);
       },
       initCommandDecorator
+      // setupCommandDecorator
```

Comment:
is this meant to be commented out?

Reply:
yes, this command is not ready hence we are not registering it

Reply:
Got it, thank you